src/components/Process/ConfirmVoteModal.tsx

```diff
@@ -88,23 +80,19 @@ const ConfirmMultiquestion = ({ question, answers }: { question: IQuestion; answ
   }
 
   return (
-    <Text display='flex' flexDirection='column' gap={1}>
-      <Trans
-        i18nKey='process.spreadsheet.confirm.options'
-        components={{
-          span: <Text as='span' fontWeight='bold' whiteSpace='nowrap' />,
-        }}
-        values={{
-          answers:
-            answers[0].length === 0
-              ? t('process.spreadsheet.confirm.blank_vote')
-              : answers[0]
-                  .map((a: string) => choices[Number(a)].title.default)
-                  .map((a: string) => `- ${a}`)
-                  .join('<br />'),
-        }}
-      />
-    </Text>
+    <Flex direction='column' gap={1}>
+      {answers[0].length === 0 ? (
+        <Text>{t('process.spreadsheet.confirm.blank_vote')}</Text>
+      ) : (
+        <UnorderedList>
+          {answers[0].map((answer: string) => (
+            <ListItem key={answer}>
+              <Text color='texts.subtle'>{choices[Number(answer)].title.default}</Text>
+            </ListItem>
+          ))}
+        </UnorderedList>
+      )}
+    </Flex>
```

Comment:
Ok so these changes make sense, but I got confused due to the name of the component which is wrong. I know you didn't put that name, but please, change it to `ConfirmMultichoice`.